Modules/SwiftRTCuda/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+add_library(SwiftRTCuda STATIC
+  compare_ops.cu
+  delayStream.cu
+  fill_ops.cu
+  math_ops.cu
+  random_ops.cu
+  reduce_ops.cu
+  specialized_ops.cu)
+target_compile_options(SwiftRTCuda PRIVATE
+  -allow-unsupported-compiler
+  --std c++17
+  --expt-relaxed-constexpr
+  -Wno-deprecated-gpu-targets
+  -gencode arch=compute_75,code=sm_75
+  --compiler-options -fPIC)
+target_include_directories(SwiftRTCuda PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR})
```

```suggestion
  -gencode=arch=compute_60,code=sm_60
  -gencode=arch=compute_70,code=sm_70
  -gencode=arch=compute_75,code=sm_75
  --compiler-options -fPIC)
target_include_directories(SwiftRTCuda PUBLIC
  ${CMAKE_CURRENT_SOURCE_DIR})

set_property(TARGET SwiftRTCuda PROPERTY CUDA_ARCHITECTURES 60 70 75)
```

This should add a broader group of CUDA compute architectures, such that we can target GTX 10XX series GPUs, and add both the new CMake 3.18 setting and a backwards-compatible fallback with the manual nvcc flags. This lets the CUDA-enabled tests build and run on my system.